### PR TITLE
PR for "Password validation rule"

### PR DESCRIPTION
**Pull request by SiddeshUbale**
This PR contains the following changes:-

1] Rule for password length validation (min 8 - max 21).
3] **dist folder** which we can refer to through Skyfall's package.json file for local testing (No need to merge into the main).

### CODE REVIEW

File: dist/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# 🌀 typhoon-ui
```
Comment (kevinmingtarja) [inline dist/README.md:1]: Can you remove all the `dist` from the changes, since we don't want to merge them.

File: package.json [hunk 2]
```diff
@@ -22,6 +22,8 @@
   },
   "homepage": "https://github.com/dgraph-io/typhoon-ui#readme",
   "dependencies": {
+    "@emotion/react": "^11.4.0",
+    "framer-motion": "^4.0.0",
```
Comment (kevinmingtarja) [inline package.json:26]: I think this shouldn't be here, since it's already there under `peerDependencies`.

**Review by kevinmingtarja — CHANGES_REQUESTED** [review]
Left some minor comments for the `dist`, `.gitignore`, and `package.json`. Otherwise everything looks good!

File: src/hooks/form/rules.ts [hunk 3]
```diff
@@ -37,4 +37,12 @@ export const RULES: Record<RuleNameType, RuleType> = {
             return value.trim();
         },
     },
+    passwordLength: {
+        type: "passwordLength",
+        message: "Password length should be between 8 to 21 characters",
```
Comment (sanjayk-github-dev) [inline src/hooks/form/rules.ts:42]: Let's use min of 8 and max of 64 characters. This matches AWS password policy.

**Review by kevinmingtarja — APPROVED** [review]
LGTM!

**Pull request merged**